### PR TITLE
fix(curriculum): Convert blockquote elements to triple backtick syntax for Information Security And Quality Assurance

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

**PR Purpose:** Converts `blockquote` elements to triple backtick syntax in the `Description` and `Instructions` sections of the `Information Security And Quality Assurance` challenges.

Related to issue #35991

### CODE REVIEW

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 1]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
...
 <section id='instructions'>
 Send <br>
-<blockquote>
-{<br>
-  &nbsp;surname: 'Colombo'<br>
-}</blockquote> 
+
+```js
+{
+  surname: 'Colombo'
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]:
```suggestion
  "surname": "Colombo"
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iv---put-method.english.md [hunk 2]
```diff
@@ -13,11 +13,8 @@ This exercise is similar to the preceding. Look at it for the details.
 
 ## Instructions
 <section id='instructions'>
-Send <br>
-<blockquote>
-{<br>
-  &nbsp;surname: 'da Verrazzano'<br>
-}</blockquote>. Replace <code>assert.fail()</code> and make the test pass.
+Send <code>{ surname: 'da Verrazzano' }.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iv---put-method.english.md]:
```suggestion
Send 

```json
{ 
  "surname": "da Verrazzano"
}
```
.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md [hunk 3]
```diff
@@ -7,9 +7,17 @@ challengeType: 2
...
+var hash = bcrypt.hashSync(myPlaintextPassword, saltRounds);
+```
+
 Add this method of hashing to your code and then log the result to the console. Again, the variables used are already defined in the server so you wont need to adjust them. You may notice even though you are hashing the same password as in the async function, the result in the console is different- this is due to the salt being randomly generated each time as seen by the first 22 characters in the third string of the hash.
-Now to compare a password input with the new sync hash, you would use the compareSync method: <blockquote>var result = bcrypt.compareSync(myPlaintextPassword, hash);</blockquote> with the result being a boolean true or false.
+Now to compare a password input with the new sync hash, you would use the compareSync method: 
+```js
+var result = bcrypt.compareSync(myPlaintextPassword, hash);
+```
+ with the result being a boolean true or false.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md]:
```suggestion

 with the result being a boolean true or false.
```
Just erring on the side of caution, by sticking a newline in.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
The whitespace problem is a little troubling, since it's not at all obvious from just looking at the diff.  Other than that it's mostly nitpicking about JSON.  If you feel anything is too nitpicky/out of scope, feel free to ignore it.

**Comment by RandellDawson** [comment]
@ojeytonwilliams I realize why this is messed up.  I had  removed the `<br>` conversions from my script when I was working on the html challenges and forgot to add it back in.  I will revert and rerun the script again before repushing.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md [hunk 4]
```diff
@@ -12,12 +12,14 @@ Strategies with OAuth require you to have at least a <em>Client ID</em> and a <e
...
-&nbsp;&nbsp;&nbsp;&nbsp;res.redirect('/profile');<br>
-&nbsp;&nbsp;});
-</blockquote>
+
+```js
+app.route('/login')
+  .post(passport.authenticate('local', { failureRedirect: '/' }), (req,res) => {
+    res.redirect('/profile');
+  });
+```
```
Comment (moT01) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: I was looking at this one... something isn't working quite right...
how does this, turn into this...
Screen Shot 2019-05-09 at 9 32 18 AM
no line breaks? 
I tried looking for spaces, adding spaces, changing things around - couldn't get it to display right

Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: @moT01 See my comment above to @ojeytonwilliams.

Comment (moT01) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: That seems like a different problem - not sure though.

**Comment by ojeytonwilliams** [comment]
Ah. That seems less than ideal, but maybe the solution will end up being to
stress that in the docs.

**Comment by RandellDawson** [comment]
@ojeytonwilliams Plus, that is why we ask the users to test things locally, to make sure everything looks OK.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md (hunk 4 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: @moT01 I see what you mean now.  @ojeytonwilliams My guess is these challenges use a different CSS file?

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/authentication-strategies.english.md [hunk 5]
```diff
@@ -10,19 +10,21 @@ As a reminder, this project is being built upon the following starter project on
...
+      if (err) { return done(err); }
+      if (!user) { return done(null, false); }
+      if (password !== user.password) { return done(null, false); }
+      return done(null, user);
+    });
+  }
 ));
-</blockquote><br>
+```
+<br>
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/authentication-strategies.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-asynchronously.english.md [hunk 6]
```diff
@@ -7,26 +7,37 @@ challengeType: 2
...
-bcrypt.hash('passw0rd!', 13, (err, hash) => {<br>
-&nbsp;&nbsp;console.log(hash);<br>
-&nbsp;&nbsp;//$2a$12$Y.PHPE15wR25qrrtgGkiYe2sXo98cjuMCG1YwSI5rJW1DSJp0gEYS<br>
-&nbsp;&nbsp;bcrypt.compare('passw0rd!', hash, (err, res) => {<br>
-&nbsp;&nbsp;&nbsp;&nbsp;console.log(res); //true<br>
-&nbsp;&nbsp;});<br>
-});<br>
-</blockquote>
+Now when you need to figure out if a new input is the same data as the hash you would just use the compare function 
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-asynchronously.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md [hunk 7]
```diff
@@ -7,9 +7,17 @@ challengeType: 2
 ## Description
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
-Hashing synchronously is just as easy to do but can cause lag if using it server side with a high cost or with hashing done very often. Hashing with this method is as easy as calling <blockquote>var hash = bcrypt.hashSync(myPlaintextPassword, saltRounds);</blockquote>
+Hashing synchronously is just as easy to do but can cause lag if using it server side with a high cost or with hashing done very often. Hashing with this method is as easy as calling 
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md [hunk 8]
```diff
@@ -7,9 +7,17 @@ challengeType: 2
...
+var hash = bcrypt.hashSync(myPlaintextPassword, saltRounds);
+```
+
 Add this method of hashing to your code and then log the result to the console. Again, the variables used are already defined in the server so you wont need to adjust them. You may notice even though you are hashing the same password as in the async function, the result in the console is different- this is due to the salt being randomly generated each time as seen by the first 22 characters in the third string of the hash.
-Now to compare a password input with the new sync hash, you would use the compareSync method: <blockquote>var result = bcrypt.compareSync(myPlaintextPassword, hash);</blockquote> with the result being a boolean true or false.
+Now to compare a password input with the new sync hash, you would use the compareSync method: 
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 9]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
 In the next example we'll see how to send data in a request payload (body).
 We are going to test a PUT request. The <code>'/travellers'</code> endpoint accepts a JSON object taking the structure :
- <blockquote>
- {<br>
-   &nbsp;surname: [last name of a traveller of the past]<br>
- } </blockquote>
+ 
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 10]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
...
+}
+```
+
 The route responds with :
- <blockquote>
- {<br>
-   &nbsp;name: [first name], surname:[last name], dates: [birth - death years]<br>
- }</blockquote>
+ 
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 11]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
...
 In the next example we'll see how to send data in a request payload (body).
 We are going to test a PUT request. The <code>'/travellers'</code> endpoint accepts a JSON object taking the structure :
- <blockquote>
- {<br>
-   &nbsp;surname: [last name of a traveller of the past]<br>
- } </blockquote>
+ 
+```js
+{
+  surname: [last name of a traveller of the past]
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 12]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
...
+
 The route responds with :
- <blockquote>
- {<br>
-   &nbsp;name: [first name], surname:[last name], dates: [birth - death years]<br>
- }</blockquote>
+ 
+```js
+{
+  name: [first name], surname:[last name], dates: [birth - death years]
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 13]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
 In the next example we'll see how to send data in a request payload (body).
 We are going to test a PUT request. The <code>'/travellers'</code> endpoint accepts a JSON object taking the structure :
- <blockquote>
- {<br>
-   &nbsp;surname: [last name of a traveller of the past]<br>
- } </blockquote>
+
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md [hunk 14]
```diff
@@ -9,26 +9,35 @@ challengeType: 2
...
 
 ## Instructions
 <section id='instructions'>
 Send <br>
-<blockquote>
-{<br>
-  &nbsp;surname: 'Colombo'<br>
-}</blockquote> 
+
+```js
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iv---put-method.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iv---put-method.english.md]: Resolved in new commit.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md (hunk 3 above)
Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md]: Resolved in new commit.

**Comment by RandellDawson** [comment]
@ojeytonwilliams and @moT01 My latest commit should have taken care of @ojeytonwilliams' requested changes.  The only issue left is what @moT01 discovered above.  Not sure what to do about that one.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md [hunk 15]
```diff
@@ -12,12 +12,14 @@ Strategies with OAuth require you to have at least a <em>Client ID</em> and a <e
 Obtaining your <em>Client ID and Secret<em> from GitHub is done in your account profile settings under 'developer settings', then '<a href='https://github.com/settings/developers'>OAuth applications</a>'. Click 'Register a new application', name your app, paste in the url to your glitch homepage (<b>Not the project code's url</b>), and lastly for the callback url, paste in the same url as the homepage but with '/auth/github/callback' added on. This is where users will be redirected to for us to handle after authenticating on GitHub. Save the returned information as 'GITHUB_CLIENT_ID' and 'GITHUB_CLIENT_SECRET' in your .env file.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]:
```suggestion
Obtaining your <em>Client ID and Secret</em> from GitHub is done in your account profile settings under 'developer settings', then '<a href='https://github.com/settings/developers'>OAuth applications</a>'. Click 'Register a new application', name your app, paste in the url to your glitch homepage (<b>Not the project code's url</b>), and lastly for the callback url, paste in the same url as the homepage but with '/auth/github/callback' added on. This is where users will be redirected to for us to handle after authenticating on GitHub. Save the returned information as 'GITHUB_CLIENT_ID' and 'GITHUB_CLIENT_SECRET' in your .env file.
```

Comment (ojeytonwilliams) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: @moT01 @RandellDawson  That's the culprit.  The `<em>` element wasn't closed and this messed up all the following instructions.

Comment (RandellDawson) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/implementation-of-social-authentication.english.md]: Oh wow! Great catch!

**Comment by RandellDawson** [comment]
@moT01 I committed @ojeytonwilliams' suggested change above and that appears to fix the problem you and I were seeing.

**Review by moT01 — APPROVED** [review]
I took another look at this locally - LGTM 🎉

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
LGTM, too :+1:

**Comment by RandellDawson** [comment]
@moT01 @ojeytonwilliams See my comment on #35993. I believe this can be merged now.

**Comment by moT01** [comment]
Nice job @RandellDawson 🎉

**Pull request merged**